### DISCUSSION TITLE
M1 맥 최신버전에서 스크립트 오픈 실패

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#503] [Issue Report (Legacy)]
M1 애플 실리콘 맥 Monterey 12.3 버전에서 스크립트 오픈에 실패

<img width="1920" alt="스크린샷 2022-03-17 오후 2 52 14" src="https://user-images.githubusercontent.com/90679926/158745682-7b3eba4b-da83-4a71-8543-540965bff20e.png">

<img width="576" alt="스크린샷 2022-03-17 오후 2 53 42" src="https://user-images.githubusercontent.com/90679926/158746033-3d4ece91-0672-4d2e-9712-74a256162cf9.png">

대체 방법

1.생성한 스크립트 폴더
2.폴더에서 더블 클릭으로 오픈
3.스크립트 수정

<img width="1255" alt="스크린샷 2022-03-17 오후 2 54 06" src="https://user-images.githubusercontent.com/90679926/158746063-fa83d50c-efc3-452a-bbfa-2ce184c331dd.png">

<img width="1255" alt="스크린샷 2022-03-17 오후 2 54 11" src="https://user-images.githubusercontent.com/90679926/158746069-7a626608-b134-4429-b934-bde16480802e.png">

<img width="1255" alt="스크린샷 2022-03-17 오후 2 54 30" src="https://user-images.githubusercontent.com/90679926/158746075-6265c2b6-10df-4af2-a9dc-f53d5cbe1ebb.png">

12.3 이전 버전에서는 정상 실행되니 아직 업데이트 하지 않으신 분들은 업데이트를 미루는것을 추천합니다.

**Comment by x4cf** [comment]
업데이트 이후 새로생성한 스크립트에 한에서 버그 발생

**Reply by x4cf** [reply]
라이더도 동일 메세지 출력

**Reply by cheolhui-lee** [reply]
저는 다행이 rider는 정상 동작을 하고 있는데요.
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/3862666/159399813-3458338f-c529-4563-a26d-f77b45d683b3.png">

rider도 동일한 에러 팝업으로 안되는걸까요?

**Reply by x4cf** [reply]
라이더도 동일한 문제가 나타나서 인텔맥으로 작업중이에요 :D
<img width="951" alt="스크린샷 2022-03-22 오후 1 54 46" src="https://user-images.githubusercontent.com/90679926/159410961-9d7bf005-def9-4a6b-a3f2-b5656357cfd2.png">
<img width="255" alt="스크린샷 2022-03-22 오후 1 55 48" src="https://user-images.githubusercontent.com/90679926/159410964-5d97a9e9-1c37-4083-9f1b-d7134357f7fa.png">

**Reply by cheolhui-lee** [reply]
넵. jetbrains toolbox를 통해 설치되어 Library 디렉토리 안에 있는 프로그램은 실행이 되는 것으로 보이고,
직접 Application으로 설치된 프로그램은 동일한 이슈가 발생하는 것 같습니다.
이 건은 애플이나 유니티에서 확인을 해줘야 할 것 같은 상황이네요.

**Comment by cheolhui-lee** [comment]
안녕하세요.
해당 내용 확인해보겠습니다.

**Reply by cheolhui-lee** [reply]
해당 내용을 확인해봤는데 이번 MacOS 12.3에서 애플리케이션을 열 때 발생하는 버그로 보입니다.
비단 유니티에서 에디터를 여는 경우가 아니더라도 다양한 곳에서 문제가 발생을 하고 있는 상황이네요.

**Reply by cheolhui-lee** [reply]
https://developer.apple.com/forums/thread/702315
https://news.i-n24.com/technology/107246.html
https://macscripter.net/viewtopic.php?pid=209763
